DonosServer/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
Raczej nie potrzenujemy przykładowego kontrolera, jak już dawałeś serwisy to zamiast tego mogłeś jakiś useful kontroler